bin/blin.p6

```diff
@@ -362,9 +363,11 @@ note ‘🥞 Saving results’;
 
 note ‘🥞🥞 Saving the overview’;
 
+my @sorted-modules;
 sub save-overview {
     $save-lock.protect: {
-        spurt $overview-path, @modules.sort(*.name).map({
+        @sorted-modules = @modules.sort(*.name);
```

Comment:
There's no need to cache sorting.

Reply:
Also, the sorting should probably be done by looking at the commit first. For example:
- [ ] Module::Bar Bisected: abc123
- [ ] Module::Baz Bisected: abc123
- [ ] Module::Foo Bisected: abc123
- [ ] Module::Bar Bisected: 311413
- [ ] Module::Baz Bisected: f3123f
- [ ] Module::Foo Bisected: f3123f

Reply:
+Well, I did that so I didn't have to write "my" over again. Are you sure you want the order of overview and overview.md to differ, or should this be done to overview, as well?

Reply:
@Xliff I think they should differ, yeah. `overview` file is just a dump of all results with each module on its own line, but `overview.md` (or maybe it should be called `rakudo-ticket.md`?) is something that is supposed to be presentable for the general public.

Reply:
`overview.md` may be a bit misleading, I'd vote for calling it `blin-round.md`, `regressions.md` or something along the lines.

Reply:
failures.md -- Author fiat! :)